src/components/vr_cooperation/src/command/base_json_request.cc

```diff
@@ -90,6 +91,20 @@ vr_hmi_api::ResultCode BaseJsonRequest::GetHMIResultCode(
   return hmiResCode;
 }
 
+functional_modules::MobileFunctionID BaseJsonRequest::GetMobileFunctionID(
+    vr_hmi_api::RPCName function_id) {
+  switch (function_id) {
+    case vr_hmi_api::ON_REGISTER:
+      return functional_modules::REGISTER_SERVICE;
+    case vr_hmi_api::ACTIVATE:
+      return functional_modules::ACTIVATE_SERVICE;
+    case vr_hmi_api::ON_DEACTIVATED:
+      return functional_modules::ON_SERVICE_DEACTIVATED;
+    default:
+      return functional_modules::PROCESS_DATA;
```

Comment:
I think in default case we should run plugin on wrong way. I mean it is better if the log has errors and warnings about sending unknown command.